backend/API/swagger_API.yaml

```diff
@@ -25,13 +24,25 @@ paths:
       parameters:
       - in: "body"
         name: "body"
-        description: "input feedback"
+        description: "feedback"
         required: true
         schema:
-          $ref: "#/definitions/Student"
+          type: "string"
```

Comment:
Also define a new object for the input that will be a JSON input instead of a string.